src/tracks/InstrumentTrack.cpp

```diff
@@ -304,6 +311,28 @@ void InstrumentTrack::processInEvent( const MidiEvent& event, const MidiTime& ti
 				}
 				else
 				{
+					if (isSustainPedalPressed())
+					{
+						for (NotePlayHandle*& nph : m_notes)
+						{
+							if (nph != NULL)
+							{
+								if (nph->isReleased())
```

Comment:
I think using && is better than using many if() clauses. In if (A && B), B isn't evaluated when A is false, maybe you know it already.
So it is safe to write like if (nph != NULL && nph->isReleased()) and that will make the code cleaner.